TEMPLATES/semgrep.yml

```diff
@@ -0,0 +1,10 @@
+#This is just an dummy rule to avoid empty file issue
+
+rules:
+- id: eqeq-is-bad
+  pattern: $X == $X
+  message: $X == $X is a useless equality check
+  languages:
+  - python
+  severity: ERROR
```

Comment:
Since this is an actual rule and no longer an empty file, let's put this at warning for now (though not sure if errors vs. warnings make a difference for super-linter, if they both end up making big angry red Xs).

Reply:
Ok good idea.